.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+ADO card: ☑️ [AB#XXXX](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Delivery%20team/Stories/?workitem=XXXX)
+
+Elab notes/AC: 📃 [Google Docs](https://docs.google.com)
```

Comment:
Perhaps something to encourage cross-team collab? e.g. "Has this been discussed with delivery team?"